microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -154,6 +154,7 @@ The following are some of the flags we've seen used.
 | `UseNativeThreadPool` | If enabled, WebView2 `ThreadPoolImpl` uses a foreground `ThreadGroup` that's backed by a native thread pool implementation. |
 | `UseBackgroundNativeThreadPool` | If this is enabled, WebView2 ThreadPoolImpl will use a background ThreadGroup backed by a native thread pool implementation. |
 | `V8Maglev` | Enables the Maglev compiler.  This only sets the V8 flag when manually overridden; otherwise, it defers to whatever the V8 default is. |
+| `BreakoutBoxPreferCaptureTimestampInVideoFrames` | Reverts the behavior of `Timestamp` property of the `Texture` interface to match legacy behavior of WebView2 runtime version 124 and earlier. |
```

Comment:
```suggestion
| `BreakoutBoxPreferCaptureTimestampInVideoFrames` | Reverts the behavior of the `Timestamp` property of the `CoreWebView2Texture` class to match legacy behavior of WebView2 Runtime version 124 and earlier. |
```
* the Texture interface - where is doc'n?  is there such an interface?  @aluhrs13 

CoreWebView2Texture Class > Timestamp property
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2texture?view=webview2-winrt-1.0.2526-prerelease#timestamp

Win32 not found: ICoreWebView2Texture - 
https://learn.microsoft.com/en-us/search/?scope=WebView2&view=webview2-1.0.2478.35&terms=ICoreWebView2Texture

* Runtime - 
full-text search in repo: 
WebView2 runtime - 5x
WebView2 Runtime - 414x